server/test/categoryRouter.test.js

```diff
@@ -146,9 +178,46 @@ describe('categoryRouter', () => {
         });
     });
   });
+  
+  describe('/getCategoryItems/:categoryId', () => {
+    it('get items in testCategory1, should return response.body.items with 2 items', (done) => {
+      chai
+        .request(server)
+        .get('/category/getCategoryItems/' + testCategory1._id)
+        .end((error, response) => {
+          expect(response).to.have.status(200);
+          expect(response.body.title).equals('User1 Category1');
```

Comment:
Nit: Maybe change this string to use the title from the variable created above (e.g. category.title or so)? You use the variable for the IDs. Feel free to ignore this if it is bad practice

Reply:
so this one isn't added from the top, so i don't have access to a variable to do .title with. But this will all change bc im re-setting up these tests